locale/fi/toast.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Ilmoitus (Toast)",
+  "intro": "Ilmoitus on komponentti, jonka avulla käyttäjälle voidaan välittää tietoa käyttäjän toimintaan liittyvää palautetta. Käytetään tilanteissa, jotka eivät vaadi käyttäjältä varsinaista reagointia. Toast-ilmoitus kertoo, että käyttäjän suorittama toimenpide onnistui.",
```

Comment:
> .. voidaan välittää **tietoa** käyttäjän toimintaan liittyvää palautetta..

Unwanted text, "tietoa". Should remove in my opinion.

Reply:
Removed it.